File: save.py
rom flask import Flask, request
import html

app = Flask(__name__)

@app.route('/', methods=['POST'])
def register():
    name = html.escape(request.form.get('name', ''))
    email = html.escape(request.form.get('email', ''))
    freefire_id = html.escape(request.form.get('freefire_id', ''))
    phone = html.escape(request.form.get('phone', ''))
    
    data = f"Name: {name}, Email: {email}, Free Fire ID: {freefire_id}, Phone: {phone}\n"
    
    with open("data.txt", "a") as file:
        file.write(data)
    
    return "Registration Successful!"

if __name__ == '__main__':
    app.run(debug=True)

File: save.php
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    $name = htmlspecialchars($_POST['name']);
    $email = htmlspecialchars($_POST['email']);
    $freefire_id = htmlspecialchars($_POST['freefire_id']);
    $phone = htmlspecialchars($_POST['phone']);

    // Change this path to match your desktop path
    $file_path = "data.txt"; // For Windows
    // $file_path = "/Users/YourUsername/Desktop/data.txt"; // For Mac/Linux

    $data = "Name: $name, Email: $email, Free Fire ID: $freefire_id, Phone: $phone\n";

    file_put_contents($file_path, $data, FILE_APPEND);

    echo "Registration Successful! Data saved to Desktop.";
} else {
    echo "Invalid request!";
}
?>
